.taskcluster.yml

```diff
@@ -4,10 +4,19 @@ policy:
 tasks:
   $if: tasks_for == "github-push"
   then:
-    $if: event.ref == "refs/heads/master"
+    $if: event.ref == "refs/heads/master" || event.ref == "refs/heads/epochs/daily" || event.ref == "refs/heads/epochs/weekly"
     then:
       $flattenDeep:
-        $map: [{name: firefox, channel: nightly}, {name: chrome, channel: dev}]
+        $map:
+          $if: event.ref == 'refs/heads/master'
+          then:
+            {config: [{name: firefox, channel: nightly}, {name: chrome, channel: dev}]}
```

Comment:
I didn't do that because it's not exhaustive, but `if`/`else` is. I agree that `$match` is more readable, but I don't know what the right tradeoff is. Maybe I shouldn't care about handling every case since hitting a case that isn't handled here resulting in a runtime error would be fine.